migration-cluster-day2/charts/migration-cluster-day2/templates/mtv-init.yaml

```diff
@@ -61,4 +61,13 @@ spec:
     kind: "Service"
     name: mtv-init
     weight: null
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vmware-credentials
```

Comment:
Why do we need empty credentials?

Reply:
Use them here https://github.com/kubev2v/migration-planner/blob/main/migration-cluster-day2/charts/migration-cluster-day2/templates/vddk.yaml#L139